tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -703,6 +706,31 @@ func (s *storeImpl) Walk(ctx context.Context, fn func(obj *{{.Type}}) error) err
 	return nil
 }
 
+{{ if eq .ResourceType "directlyScoped" }}
+func isInScope(obj *{{.Type}}, eas effectiveaccessscope.ScopeTree) bool {
+    if eas.State == effectiveaccessscope.Included {
+        return true
+    }
+    if eas.State == effectiveaccessscope.Excluded {
+        return false
+    }
+    clusterId := {{ .ClusterGetter }}
+    cluster := eas.Clusters[clusterId]
```

Comment:
You probably want to call `eas.GetClusterByID(clusterId)` here.
The effectiveaccessscope ScopeTree structure indexes clusters by name rather than by ID